### PR TITLE
Frist draft of the ACROSS landing page

### PR DESCRIPTION
**Pull request by dankocevski**
First draft of the ACROSS landing page.  All elements are USWDS components with some additional styling, except for the ParallaxImage component.  Inline styling will eventually be replaced with custom style sheets as needed, but left in place to make it clear which components utilize custom css. The content in card components will ultimately be stored in a markdown format.

### CODE REVIEW

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const handleScroll = () => {
+    const newOffset = window.pageYOffset
+    setOffset(newOffset)
+  }
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll)
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [])
+
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:21]: Did you copy this from somewhere? If so, it would be preferable to add it to package.json as a dependency. If that's not possible, then we need to reproduce any relevant license information.

Also, it would be preferable to organize this as a reusable component, rather than hardcoding its content.

File: app/routes/labs/route.tsx [hunk 2]
```diff
@@ -25,10 +25,7 @@ export default function () {
   return (
     <>
       <Header />
-      <h1>ACROSS</h1>
-      <main>
-        <Outlet />
-      </main>
```
Comment (lpsinger) [inline app/routes/labs/route.tsx:31]: Why did you remove the main section?

File: app/routes/labs._index.mdx [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+          alt="Descriptive Alt Text"
+          style={{
+            width: 'auto',
+            height: '600px',
+            marginLeft: '0px'
+          }}
+        />
+      </div>
+
+      <div className="section-head" style={{marginTop:'150px'}}></div>
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:81]: Why is there an empty div?

File: app/routes/labs._index.mdx [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+} from '@trussworks/react-uswds'
+
+import ParallaxImage from './labs/parallax/ParallaxImage'
+
+import React from 'react'
+
+import styles from './labs/across.css'
+import mma from './labs/mma.png'
+
+  <div style={{ backgroundColor: '#0c0f18', color: 'white', minHeight: '2000px' }}>
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:29]: Is every page going to have a different background color and height? If not, then these styles should go in the layout route.

File: app/routes/labs._index.mdx [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+
+    <ParallaxImage
+      src="https://svs.gsfc.nasa.gov/vis/a010000/a012700/a012740/NeutronStarMerger_Shot1_4KStill4_v01_print.jpg"
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:32]: Please add these images to the source tree and import them, so that they are bundled and distributed over our content delivery network and delivered to visitors from edge locations close to them.

File: app/routes/labs._index.mdx [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+      alt="BNS"
+    />
+
+    <GridContainer className="usa-section" style={{minWidth:'75%'}}>
+
+      <div className="section-head"><h1>Overview</h1></div>
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:38]: I don't see the `section-head` class defined anywhere.

File: app/routes/labs._index.mdx [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+
+      <div className="container" style={{ marginTop: '0px', display: 'flex', justifyContent: 'space-between' }}>
+        <div className="usa-paragraph text-base-lightest magin-top-0" style={{
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:41]: You can remove the `use-paragraph` style because we apply it automatically to all paragraphs in Markdown and MDX files: https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/remix.config.js#L53

```suggestion
        <div className="text-base-lightest magin-top-0" style={{
```

File: app/routes/labs._index.mdx [hunk 8]
```diff
@@ -0,0 +1,293 @@
...
+            fontSize: '1.1rem',
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:42]: Is this supposed to make these paragraphs slightly larger than body text? If so, then please use the `usa-intro` utility class.

File: app/routes/labs._index.mdx [hunk 9]
```diff
@@ -0,0 +1,293 @@
...
+            zIndex: 1,
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:43]: Why is it necessary to adjust the z-ordering of these paragraphs? Are they being occluded by something else on the page?

File: app/routes/labs._index.mdx [hunk 10]
```diff
@@ -0,0 +1,293 @@
...
+            whiteSpace: 'normal',
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:44]:
```suggestion
```

File: app/routes/labs._index.mdx [hunk 11]
```diff
@@ -0,0 +1,293 @@
...
+            width: '90%',
+            marginTop: '0px'
+
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:47]: Unnecessary whitespace.

```suggestion
```

File: app/routes/labs._index.mdx [hunk 12]
```diff
@@ -0,0 +1,293 @@
...
+      <div className="section-head"><h1>Overview</h1></div>
+
+      <div className="container" style={{ marginTop: '0px', display: 'flex', justifyContent: 'space-between' }}>
+        <div className="usa-paragraph text-base-lightest magin-top-0" style={{
+            fontSize: '1.1rem',
+            zIndex: 1,
+            whiteSpace: 'normal',
+            width: '90%',
+            marginTop: '0px'
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:46]: Why is manual sizing necessary here?

File: app/routes/labs._index.mdx [hunk 13]
```diff
@@ -0,0 +1,293 @@
...
+
+
+          Multimessenger.nasa.gov aims to support this emerging field by consolidating important information on resources available to
+          scientists around the world to enable future discoveries.
+
+        </div>
+
+        <img
+          src={mma}
+          alt="Descriptive Alt Text"
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:72]: This looks like placeholder text.

File: app/routes/labs._index.mdx [hunk 14]
```diff
@@ -0,0 +1,293 @@
...
+          headerFirst
+          gridLayout={{ tablet: { col: 5 } }}
+          containerProps={{ className: '', style: { borderRadius: '10px', borderColor: '#161821' } }}
+          >
+          <CardBody className="padding-4 text-base-lightest" style={{ backgroundColor:'#161821'}}>
+            <h1 className="text-left" style={{paddingTop: '20px'}}>
+              The Decadal Survey Findings on Multi-messenger Astronomy
+            </h1>
+          </CardBody>
+        </Card>
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:95]: There's a lot of custom styling here.

- [ ] Please use [theme color tokens](https://designsystem.digital.gov/design-tokens/color/theme-tokens/) rather than hardcoded hexadecimal colors. Theme colors give us a consistent look and feel across the site.
- [ ] Do these custom styles define how you would like cards to appear in dark mode throughout the ACROSS site? If so, then we should move this to the [USWDS theme customization](https://designsystem.digital.gov/documentation/settings/), which probably also entails emitting two versions of the USWDS CSS, one for GCN and one for ACROSS. If not, then please factor these custom styles into a React component defined in this route.

File: app/routes/labs._index.mdx [hunk 15]
```diff
@@ -0,0 +1,293 @@
...
+
+        <Card
+          headerFirst
+          gridLayout={{ tablet: { col: 7 } }}
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:99]: It looks like every card has a different width. Is that intentional?

File: app/routes/labs._index.mdx [hunk 16]
```diff
@@ -0,0 +1,293 @@
+{/\* \* Copyright © 2023 United States Government as represented by the
+
+- Administrator of the National Aeronautics and Space Administration.
+- All Rights Reserved.
+-
+- SPDX-License-Identifier: Apache-2.0 \*/}
+
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:7]: We've been omitting the license header from MDX files.

```suggestion
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/labs/parallax/ParallaxImage.tsx (hunk 1 above)
Comment (dankocevski) [reply app/routes/labs/parallax/ParallaxImage.tsx:21]: Nope, this was my own creation after looking at how various other image related components were structured.  

I've restructured the code to allow the title, paragraph text, and image to be specified as properties to make it a reusable component.

File: app/routes/labs/route.tsx (hunk 2 above)
Comment (dankocevski) [reply app/routes/labs/route.tsx:31]: I didn't see an equivalent main section in other index pages. Including one also added a white border below the header.

File: app/routes/labs._index.mdx (hunk 3 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:81]: I decided to have a section header with an optional section title that would separate each major segment of the page. The contents of each segment could easily be moved into the section instead.

File: app/routes/labs/route.tsx (hunk 2 above)
Comment (lpsinger) [reply app/routes/labs/route.tsx:31]: The reason that this page has a `<main>` element and most others don't is that this is the layout route for all pages under /labs. The `<main>` element is required for accessibility. The white border is a problem; I will do some research to determine how to tell the browser to prefer dark mode, which is supported by USWDS.

File: app/routes/labs._index.mdx (hunk 3 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:81]: I've gone ahead and created the same effect by using Grid rows rather than custom sections.

File: app/routes/labs._index.mdx (hunk 4 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:29]: I've made this into a layout component that is defined in the ./labs/routes.tsx file.

File: app/routes/labs._index.mdx (hunk 5 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:32]: Done.

File: app/routes/labs._index.mdx (hunk 6 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:38]: Indeed, these were defined in the across.css file, but never actually used, so I've removed the class reference. In any case, these section headers have now been replaced by Grid rows.

File: app/routes/labs._index.mdx (hunk 7 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:41]: Done.

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+        src={src}
+        alt={alt}
+        style={{
+          position: 'absolute',
+          width: '100%',
+          top: '-10%',
+          bottom: '-5%',
+          transform: `translateY(${offset * 0.5}px)`,
+        }}
+      />
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:60]: Please separate out this content so that `ParallaxImage` is a reusable component that you can employ like this:

```jsx
<ParallaxImage>
  <h3>The Dawn of a New Era in Astronomy</h3>
  <p className="usa-paragraph">Lorem ipsum...</p>
</ParallaxImage>
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+    </div>
+  )
+}
+
+export default ParallaxImage
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:65]: We tend not to have default exports in library files.

```suggestion
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 19]
```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react'
+
+// Define an interface for the component's props
+interface ParallaxImageProps {
+  src: string // URL of the image
+  alt: string // Description of the image
+}
+
+function ParallaxImage({ src, alt }: ParallaxImageProps) {
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:9]:
```suggestion
export function ParallaxImage({ src, alt }: ParallaxImageProps) {
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 20]
```diff
@@ -0,0 +1,65 @@
...
+
+  const handleScroll = () => {
+    const newOffset = window.pageYOffset
+    setOffset(newOffset)
+  }
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll)
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [])
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:20]: Does [usehooks-ts](https://usehooks-ts.com) supply any React hooks for watching updates to the scroll position? We already have a dependency on it.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/labs._index.mdx (hunk 8 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:42]: usa-intro is a slightly larger than what I'm aiming for here, but we could also consider reducing the amount of text that goes into this intro.

Screen Shot 2023-12-20 at 4 21 59 PM
-0558-4163-8701-cb9d62ef1458">

File: app/routes/labs._index.mdx (hunk 9 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:43]: No reason in final implementation, so removed.

File: app/routes/labs._index.mdx (hunk 10 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:44]: No reason this needs to be in there in final implementation, so its been removed.

File: app/routes/labs._index.mdx (hunk 11 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:47]: Removed

File: app/routes/labs._index.mdx (hunk 12 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:46]: This is done to add additional margin between the text block and the image to the right. The same affect could be implemented with a grid column if necessary.

File: app/routes/labs._index.mdx (hunk 13 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:72]: Text has been replaced.

File: app/routes/labs._index.mdx (hunk 15 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:99]: Yes, the block quote source and text cards span 5 and 7 columns respectively, while the program and science news each span 4 columns each.

File: app/routes/labs._index.mdx (hunk 16 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:7]: Removed.

File: app/routes/labs._index.mdx (hunk 8 above)
Comment (lpsinger) [reply app/routes/labs._index.mdx:42]: What about making this paragraph sorter? It's a pretty long paragraph.

File: app/routes/labs._index.mdx (hunk 16 above)
Comment (lpsinger) [reply app/routes/labs._index.mdx:7]: It's still there.

File: app/routes/labs._index.mdx (hunk 9 above)
Comment (lpsinger) [reply app/routes/labs._index.mdx:43]: It's still there.

File: app/routes/labs._index.mdx (hunk 7 above)
Comment (lpsinger) [inline app/routes/labs._index.mdx:41]:
```suggestion
        <div className="text-base-lightest magin-top-0" style={{
```

File: app/routes/labs._index.mdx (hunk 16 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:7]: I haven't pushed these changes yet.

File: app/routes/labs/parallax/ParallaxImage.tsx (hunk 17 above)
Comment (dankocevski) [reply app/routes/labs/parallax/ParallaxImage.tsx:60]: Done.

File: app/routes/labs/parallax/ParallaxImage.tsx (hunk 18 above)
Comment (dankocevski) [reply app/routes/labs/parallax/ParallaxImage.tsx:65]: Done.

File: app/routes/labs/parallax/ParallaxImage.tsx (hunk 19 above)
Comment (dankocevski) [reply app/routes/labs/parallax/ParallaxImage.tsx:9]: Done.

Comment (lpsinger) [reply app/routes/labs/parallax/ParallaxImage.tsx:9]: To save some confusion, would you please hold off on marking a comment as done until you've pushed the changes? It will save me the trouble of re-reviewing your PR until after I can see the changes. Thanks!

File: app/routes/labs/parallax/ParallaxImage.tsx (hunk 20 above)
Comment (dankocevski) [reply app/routes/labs/parallax/ParallaxImage.tsx:20]: Yes, useEventListener from usehooks-ts let me accomplish the same thing, so I've replaced the code.

File: app/routes/labs._index.mdx (hunk 7 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx:41]: Done.

File: app/routes/labs._index.mdx (hunk 14 above)
Comment (dankocevski) [reply app/routes/labs._index.mdx]: The closest theme color tokens to the current card background color didn't match well with the page background color (which is slightly off black), so I've left the hexadecimal colors in there for now.  I'll investigate a page background and card background combination using only the theme colors for all of the ACROSS pages and update later.

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 21]
```diff
@@ -0,0 +1,77 @@
...
+    setOffset(newOffset)
+  }
+
+  // Using useEventListener hook
+  useEventListener('scroll', handleScroll)
+
+  return (
+    <div style={containerStyle}>
+      <div style={textStyle}>
+        {children} {/* Use children instead of specific title and paragraph */}
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:65]:
```suggestion
        {children}
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+  children, // Add children here
+}: ParallaxImageProps) {
+  const [offset, setOffset] = useState(0)
+
+  const handleScroll = () => {
+    const newOffset = window.pageYOffset
+    setOffset(newOffset)
+  }
+
+  // Using useEventListener hook
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:59]: This comment is tautological:

```suggestion
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 23]
```diff
@@ -0,0 +1,77 @@
...
+  bottom: '-5%',
+}
+
+export function ParallaxImage({
+  src,
+  alt,
+  containerStyle = defaultContainerStyle,
+  textStyle = defaultTextStyle,
+  imageStyle = defaultImageStyle,
+  children, // Add children here
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:50]:
```suggestion
  children,
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 24]
```diff
@@ -0,0 +1,77 @@
...
+  top: '10%',
+  left: '29%',
+  transform: 'translateX(-50%)',
+  width: '500px',
+  fontSize: '1.25rem',
+  zIndex: 1,
+  whiteSpace: 'normal',
+}
+
+// Define a default image style
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:36]:
```suggestion
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 25]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+// Define a default container style
+const defaultContainerStyle: CSSProperties = {
+  overflow: 'hidden',
+  position: 'relative',
+  height: '800px',
+}
+
+// Define a default text style
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:24]:
```suggestion
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 26]
```diff
@@ -0,0 +1,77 @@
...
+  alt: string
+  title?: string
+  paragraph?: string
+  containerStyle?: CSSProperties
+  textStyle?: CSSProperties
+  imageStyle?: CSSProperties
+  children?: ReactNode
+}
+
+// Define a default container style
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:17]:
```suggestion
```

File: app/routes/labs/parallax/ParallaxImage.tsx [hunk 27]
```diff
@@ -0,0 +1,77 @@
+import type { CSSProperties, ReactNode } from 'react'
```
Comment (lpsinger) [inline app/routes/labs/parallax/ParallaxImage.tsx:1]: Add missing license header.

File: app/routes/labs._index.mdx [hunk 28]
```diff
@@ -0,0 +1,294 @@
...
+import './labs/across.css'
+import mma from './labs/mma.png'
+
+    <ParallaxImage src={neutronStarMerger} alt="BNS">
+      <h3>The Dawn of a New Era in Astronomy</h3>
+      <p>Multimessenger and time-domain astrophysics are opening entirely new
+          windows on the Universe. The Astrophysics Cross-Observatory Science
+          Support (ACROSS) center is an initiative to facilitate communication,
+          coordination, and collaboration in this new era of astronomy.</p>
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:29]:
```suggestion
      ### The Dawn of a New Era in Astronomy

      Multimessenger and time-domain astrophysics are opening entirely new
      windows on the Universe. The Astrophysics Cross-Observatory Science
      Support (ACROSS) center is an initiative to facilitate communication,
      coordination, and collaboration in this new era of astronomy.
```

File: app/routes/labs._index.mdx [hunk 29]
```diff
@@ -0,0 +1,294 @@
...
+
+          <Card
+            gridLayout={{ tablet: { col: 4 } }}
+            containerProps={{style: { borderRadius: '10px', borderWidth: '0px', backgroundColor: '#161821' } }}
+            >
+            <CardHeader>
+              <h3 className="usa-card__heading text-base-lightest">NASA Selects Proposals to Study Stellar Explosions, Galaxies, Stars</h3>
+            </CardHeader>
+            <CardMedia>
+              <img src="https://svs.gsfc.nasa.gov/vis/a010000/a012700/a012740/NeutronStarMerger_Shot1_4KStill4_v01_print.jpg" alt="" />
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:263]: Add this image to the repository.

File: app/routes/labs._index.mdx [hunk 30]
```diff
@@ -0,0 +1,294 @@
...
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:281]: Add this image to the repository.

File: app/routes/labs._index.mdx [hunk 31]
```diff
@@ -0,0 +1,294 @@
...
+            <CardFooter className="padding-top-3" style={{ backgroundColor:'#161821'}}>
+              <Button type="button" outline>Link</Button>
+            </CardFooter>
+          </Card>
+
+        </CardGroup>
+      </Grid>
+
+      <Grid row style={{marginTop:'75px'}}>
+        <div><h1>Science News</h1></div>
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:176]:
```suggestion
        # Science News
```

File: app/routes/labs._index.mdx [hunk 32]
```diff
@@ -0,0 +1,294 @@
...
+          windows on the Universe. The Astrophysics Cross-Observatory Science
+          Support (ACROSS) center is an initiative to facilitate communication,
+          coordination, and collaboration in this new era of astronomy.</p>
+    </ParallaxImage>
+
+    <GridContainer className="usa-section" style={{minWidth:'75%'}}>
+
+      <Grid row style={{marginTop:'20px'}}>
+
+        <div><h1>Overview</h1></div>
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:36]:
```suggestion
        # Overview
```

File: app/routes/labs._index.mdx [hunk 33]
```diff
@@ -0,0 +1,294 @@
...
+  Link,
+  Tag,
+} from '@trussworks/react-uswds'
+
+import React from 'react'
+
+import neutronStarMerger from './labs/NeutronStarMerger_Shot1_4KStill4_v01_print.jpg'
+import { ParallaxImage } from './labs/parallax/ParallaxImage'
+
+import './labs/across.css'
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:21]: Missing file.

File: app/routes/labs._index.mdx [hunk 34]
```diff
@@ -0,0 +1,294 @@
...
+
+          <Card
+            gridLayout={{ tablet: { col: 4 } }}
+            containerProps={{style: { borderRadius: '10px', borderWidth: '0px', backgroundColor: '#161821' } }}
+            >
+            <CardHeader>
+              <h3 className="usa-card__heading text-base-lightest">NASA Selects Proposals to Study Stellar Explosions, Galaxies, Stars</h3>
+            </CardHeader>
+            <CardMedia>
+              <img src="https://svs.gsfc.nasa.gov/vis/a010000/a012700/a012740/NeutronStarMerger_Shot1_4KStill4_v01_print.jpg" alt="" />
```
Comment (lpsinger) [inline app/routes/labs._index.mdx:206]: Add image to repository. Missing alt text.

**Review by lpsinger — CHANGES_REQUESTED** [review]
Some overall issues:

- Maintainability: there are a lot of inline styles and non-semantic markup. Some of this is to invert the color scheme; to achieve that, I suggest adding global styling to theme.scss:

```scss
.usa-prose--dark {
  .usa-card {
    ... // customize here
   }

  // etc.
}
```

- Accessibility: missing alt text for images.
- Performance: images should be added to the repository and imported so that they are fingerprinted and cached by the CDN.

**Comment by lpsinger** [comment]
Just wanted to check on this PR again as it's been 3 weeks since the last correspondence on it. Are you going to address the remaining comments and fix the merge conflicts?

**Comment by lpsinger** [comment]
Should we close this now that @Kirill-Vorobyev has taken it over?
